Makefile

```diff
@@ -207,7 +209,7 @@ codecheck: test/static-code $(NODE_MODULES_TEST)
 
 # convenience target to setup all the bits needed for the integration tests
 # without actually running them
-prepare-check: $(NODE_MODULES_TEST) $(VM_IMAGE) test/common
+prepare-check: $(NODE_MODULES_TEST) vm test/common
```

Comment:
Please don't depend on a phony target. This was deliberately done with `VM_IMAGE` so that repeated `make checks` *don't* always rebuild the VM (which is annoying). I.e. revert this and the hunk above.

Reply:
This will depend on the `.PHONY` target, which will always call `make` on the `$(VM_IMAGE)` target. but that will be a no-op if nothing changed.  I tested this.  :)

Reply:
I added a note about this to the commit message.

Reply:
So what's the advantage of doing it like this? It feels weird that you *introduce* an extra (useless) `make` call in a PR which is supposed to make things faster.

Reply:
It's about giving me a way to set the environment variable.